test/e2e/iptables-vpc-nat-gw/e2e_test.go

```diff
@@ -34,6 +38,34 @@ const vpcNatGWConfigMapName = "ovn-vpc-nat-gw-config"
 const networkAttachDefName = "ovn-vpc-external-network"
 const externalSubnetProvider = "ovn-vpc-external-network.kube-system"
 
+const iperf2Port = "20288"
+const skipIperf = true
```

Comment:
Iperf will cause the program to spend more time on e2e, taking approximately 15 minutes.
I will try to optimize the program to reduce the time to 5 minutes and then enable the iperf test.